package.json

```diff
@@ -58,13 +59,14 @@
     "lint": "eslint katex.js server.js cli.js webpack.config.js webpackDevServer.js src test contrib dockers && stylelint static/fonts.less static/katex.less",
     "flow": "flow",
     "jest": "jest",
+    "jest-update": "jest --updateSnapshot",
     "coverage": "jest --coverage",
     "copy": "cp -a static/. build/ && cp -a contrib build/",
     "clean": "rm -rf build/* node_modules/",
     "clean-install": "npm run clean && npm i",
     "test": "check-dependencies && npm run lint && npm run flow && npm run jest",
     "build-css": "lessc --clean-css static/katex.less build/katex.css",
-    "prestart": "npm run build-css && npm run copy",
+    "prestart": "rimraf build && npm run build-css && npm run copy",
```

Comment:
What was happening if `build` was still around?

Reply:
https://stackoverflow.com/questions/20705677/mv-cannot-overwrite-directory-with-non-directory

Couldn't overwrite a directory with the same name.